src/de/darmstadt/tu/crossing/tink/SignatureKeyTemplates.cryptsl

```diff
@@ -0,0 +1,36 @@
+/*
+ * This is a CrySL specification for the Google Tink implementation of 
+ * the class SignatureKeyTemplates. 
+ * 
+ * Note: in this case, we are not specifying the correct usage of the 
+ * API, but actually guaranteeing that the Google Tink API implementation 
+ * actually exposes some key templates as static attributes. This 
+ * information simplifies the specification of other properties for 
+ * correctly using the API. 
+ * 
+ */
+
+SPEC com.google.crypto.tink.signature.SignatureKeyTemplates
+
+OBJECTS
+	com.google.crypto.tink.proto.KeyTemplate ecdsa;
+	
+	com.google.crypto.tink.proto.HashType hashType;
+	com.google.crypto.tink.proto.EllipticCurveType curveType;
+	
+EVENTS
+	ecdsa_evt : ecdsa = createEcdsaKeyTemplate(hashType, curveType, _);
+	
+	CreateKeyTemplates := ecdsa_evt;
+	
+ORDER 
+	(CreateKeyTemplates)
+
+//CONSTRAINTS

```

Comment:
Do you not need these constraints? If so, please delete them.

Reply:
These constraints refer to enumerated types. I decided to keep this information using commented lines because future versions of CrySL might support the definition of constraints to enumerated types.